front/src/applications/operationalStudies/components/MacroEditor/osrdToNge.ts

```diff
@@ -244,12 +244,16 @@ const importTimetable = async (
   );
   const trainSchedules = await trainSchedulesPromise.unwrap();
 
+  const filteredTrainSchedules = trainSchedules.filter(
```

Comment:
Nit: I think I'd prefer to make `trainSchedules` mutable (with `let`) and overwrite it instead of using a separate `const`, otherwise I'm worried new code would forget about this stuff and use `trainSchedules` instead of `filteredTrainSchedules`.

Reply:
You're right, makes more sense keeping trainSchedules ! modified it 👍